src/playlist-controller.js

```diff
@@ -2162,6 +2162,15 @@ export class PlaylistController extends videojs.EventTarget {
     if (!currentPathway) {
       return;
     }
+
+    const pastClones = this.contentSteeringController_.currentPathwayClones;
```

Comment:
I'm wondering if it encapsulate things a bit better to move this into the `handlePathwayClones_` function and do something like `if (!hasClones) return` and just call `handlePathwayClones_` regardless?

Reply:
good call, moved that to the handlePathwayClones_ function in my most recent commit